pkg/tests/integration/test_salt_grains.py

```diff
@@ -39,3 +39,12 @@ def test_grains_setval_key_val(salt_cli, salt_minion):
     ret = salt_cli.run("grains.setval", "key", "val", minion_tgt=salt_minion.id)
     assert ret.data, ret
     assert "key" in ret.data
+
+
+def test_grains_package_onedir(salt_cli, salt_minion):
+    """
+    Test that the package grain returns onedir
+    """
+    ret = salt_cli.run("grains.get", "package", minion_tgt=salt_minion.id)
+    assert "onedir" == ret.data
```

Comment:
```suggestion
    assert ret.data == "onedir"
```
Assert what we get against what we expect. Looks more logical in case of failures on pytest diff's